### PR TITLE
add arabica instructions for Full Consensus Node, resolves #912

### LINKED ISSUE #912: Arabica tab for full consensus node setup the p2p networks
## Context

I'm trying to set up a full consensus node on Arabica

## Problem

There is no Arabica tab on https://docs.celestia.org/nodes/full-consensus-node/#setup-the-p2p-networks

## Proposal

Add an Arabica tab

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Closes #912 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by rootulp** [comment]
I think PR Preview failed to deploy [here](https://github.com/celestiaorg/docs/actions/runs/5714459267/job/15481888059?pr=925#step:4:40) b/c some links still use the old filename. Perhaps they need to reference the `.mdx` file now?

**Comment by jcstein** [comment]
yes, looking into this and thank you for flagging

File: docs/nodes/full-consensus-node.mdx [hunk 1]
```diff
@@ -184,30 +194,30 @@ copied from.
...
-    egrep -o ">blockspacerace.*tar" | tr -d ">")
-wget -O - https://snaps.qubelabs.io/celestia/${SNAP_NAME} | tar xf - \
-    -C ~/.celestia-app/data/
-```
+<pre><code>
+cd $HOME<br/>
+rm -rf ~/.celestia-app/data<br/>
+mkdir -p ~/.celestia-app/data<br/>
+SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
+    egrep -o ">{constants.arabicaChainId}.*tar" | tr -d ">")<br/>
```
Comment (jcstein) [inline docs/nodes/full-consensus-node.mdx:217]: @mindstyle85 - do you all have a snapshot hosted for this? if not, @Bidon15 we may need to run our own and i will edit this accordingly

Comment (qubelabsio) [reply docs/nodes/full-consensus-node.mdx:217]: > @qubelabsio - do you all have a snapshot hosted for this? if not, @Bidon15 we may need to run our own and i will edit this accordingly

@jcstein both are now maintained at https://snaps.qubelabs.io/celestia/, so just use the `mocha-3` and `arabica-9` tags.

Comment (jcstein) [reply docs/nodes/full-consensus-node.mdx:217]: amazing, thank you!

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**